app/models/github_adapter.rb

```diff
@@ -0,0 +1,66 @@
+class GithubAdapter
+  def initialize(post)
+    @post = post
+  end
+
+  def create_pr
+    new_branch_sha = create_branch(master_sha)
```

Comment:
It will, but in this case there is no need to make the extra call to get the ref, since we already get it in the response. I can maybe rename this to `branch_sha`, which will shadow the method, but maybe is more clear that they're the same.